Plugins/Public/mobiledocking_plugin/MobileDocking Plugin.vc14.vcxproj

```diff
@@ -132,7 +132,6 @@ copy /Y $(OutDir)$(TargetName).pdb  ..\..\..\Binaries\bin-vc14\flhook_plugins\pu
   <ItemGroup>
     <ClCompile Include="ClientCommands.cpp" />
     <ClCompile Include="Main.cpp" />
-    <ClCompile Include="SaveFiles.cpp" />

```

Comment:
I think we should list the pros and cons of removing that functionality.

Reply:
We sort of discussed this in PM but I'd like to see a more formal justification laid out on the PR

Reply:
About why I removed SaveFiles.cpp.
1) That thing is built on client ID to assign loaded data from file to that ID, what is absolutely totally wrong, because client ID is temporary variable. And when player enters the server next time, it may be (actually in 99% cases) not same.
2) Serializing seems to be not complete, it does its job well in half of cases, if turn blind eye on 1), but other half will generate bugs(plugin tells you that you have some ships on board, when actually not), which seem to be critical with new jettisoning mechanic. It removes ship only AFTER player was kicked and player remains docked to carrier until being kicked. Carrier is not able to jettison ship, which is not docked to him.